doc/imageview_warmups.md

```diff
@@ -0,0 +1,329 @@
+If you're new to Reactive or functionally-reactive programming in
+general, it pays to spend some time exploring Reactive's programming
+paradigm.  This documents a set of experiments conducted while
+rewriting ImageView.jl using Reactive and Gtk; perhaps they will serve
+as useful demonstrations for others.
+
+## Core concept: deferred execution
+
+Actions that you take on Reactive nodes are `push!`ed to a queue and
+run at some later time:
+
+```jl
+julia> using Reactive
+
+julia> n = Node(0)
+Node{Int64}(0, nactions=0)
+
+julia> push!(n, 1)
+
+julia> value(n)
+0
+
+julia> Reactive.run_till_now()
+
+julia> value(n)
+1
+```
+
+You can cause these updates to be processed at regular intervals---if
+julia isn't busy doing something else---as follows:
+```jl
+eventloop = Timer(_->Reactive.run_till_now(), 0.001, 0.001)
```

Comment:
We should definitely update this for the new changes and merge this. This will be very useful for library writers using Reactive.